backend/de.metas.handlingunits.base/src/main/java/de/metas/product/impexp/ImportProductInterceptor.java

```diff
@@ -37,21 +42,38 @@
 import org.adempiere.model.InterfaceWrapperHelper;
 import org.compiere.model.I_C_UOM;
 import org.compiere.model.I_I_Product;
-import org.springframework.stereotype.Service;
 
 import java.math.BigDecimal;
 
-@Service
-public class HUProductImportProcessListener implements ProductImportProcessListener
+/**
+ * @author metas-dev <dev@metasfresh.com>
+ */
+public class ImportProductInterceptor implements IImportInterceptor
 {
 	@NonNull private final IHandlingUnitsDAO handlingUnitsDAO = Services.get(IHandlingUnitsDAO.class);
 	@NonNull private final IUOMDAO uomDAO = Services.get(IUOMDAO.class);
 
-	public void afterRecordImport(@NonNull final I_I_Product importRecord)
+	public static final ImportProductInterceptor instance = new ImportProductInterceptor();
```

Comment:
pls don't create a static instance when u have services that you initialize on construction time.
pls create the instance when u register the interceptor.